helix-sitemap.yaml

```diff
@@ -565,3 +565,33 @@ sitemaps:
         alternate: /br/{path}.html
         destination: /br/cc-shared/assets/sitemap.xml
         hreflang: pt-BR  
+      
+      cis_en:
+        source: /cis_en/cc-shared/assets/query-index.json
+        alternate: /cis_en/{path}.html
+        destination: /cis_en/cc-shared/assets/sitemap.xml
+        hreflang:
+          - en-AM
+          - en-AZ
+          - en-BY
+          - en-KZ
+          - en-KG
+          - en-MD
+          - en-RU
+          - en-TJ
+          - en-UZ
+
+      cis_ru:
+        source: /cis_ru/cc-shared/assets/query-index.json
+        alternate: /cis_ru/{path}.html
+        destination: /cis_ru/cc-shared/assets/sitemap.xml
+        hreflang:
+          - ru-AM
+          - ru-AZ
+          - ru-BY
+          - ru-KZ
+          - ru-KG
+          - ru-MD
+          - ru-RU
+          - ru-TJ
+          - ru-UZ
```

Comment:
Please update the hreflang as shared by Sharon
hreflang=“ru-by” << cis_ru
hreflang=“en-by” << cis_en

<img width="379" alt="Screen Shot 2024-03-05 at 9 11 55 PM" src="https://github.com/adobecom/cc/assets/69535463/5795b115-bdee-4e7b-bccc-6814996b63cc">

Reply:
Also for sitemap for new geos we'll need to update helix-query.yaml too 
And make sure query-index.xlsx is set, previewed and published.

As these are new geos once the PR is merged sitemaps will need to be generated using api and then those sitemap links need to be added to index file here during go live https://github.com/adobecom/cc/blob/stage/creativecloud/sitemap-index.xml

Reply:
thx @Ruchika4 I updated the PR and also created index files for cis_ru and cis_en:

[cis_en](https://adobe.sharepoint.com/sites/adobecom/CC/Forms/AllItems.aspx?newTargetListUrl=%2Fsites%2Fadobecom%2FCC&viewpath=%2Fsites%2Fadobecom%2FCC%2FForms%2FAllItems%2Easpx&id=%2Fsites%2Fadobecom%2FCC%2Fwww%2Fcis%5Fen%2Fcc%2Dshared%2Fassets&viewid=820c5cd2%2Dfd43%2D4244%2D92c4%2Dbe6ed86f524e)